language/Turkish/strings.po

```diff
@@ -2423,24 +2427,24 @@ msgstr ""
 
 #: src/frontend/mame/ui/submenu.cpp:108
 msgid "Bitmap Prescaling"
-msgstr ""
+msgstr "Bitmap ön oranlama"
 
 #: src/frontend/mame/ui/submenu.cpp:109
 msgid "Window Mode"
-msgstr ""
+msgstr "Pencere Kipi"
 
 #: src/frontend/mame/ui/submenu.cpp:110
-msgid "Enforce Aspect Ratio"
-msgstr ""
+  msgid "Enforce Aspect Ratio"
```

Comment:
There seems to be a stray character added to the beginning of this line.  Could you please correct it?

Reply:
deleted the stray character